src/Morphir/IR/SDK/Basics.elm

```diff
@@ -40,6 +40,7 @@ moduleSpec =
     { types =
         Dict.fromList
             [ ( Name.fromString "Int", OpaqueTypeSpecification [] |> Documented "Type that represents an integer value." )
```

Comment:
@AttilaMihaly Does this seem fine to keep?

Reply:
I think it would be confusing to keep it since `Int` is really `Int32`. I think we should have only one way of doing things in the SDK.

Reply:
I found that quite a few of the tests fail when I remove `Int` is there another mapping I need to add, or do I need to change all the tests?

Reply:
There's an `intType` helper function in this module. Changing that should fix most of those tests.